### PR TITLE
Refactor/further refactor botz game

### PR DESCRIPTION
**Pull request by hoomberto**
# Description

Updated BotZ explanation text to have a more succinct explanation of game. Chunked down announce, checkOpponentHealth, selectSign, and compareQualities. Rearranged functions to make more sense relative to where they're now around, deleted commeneted out lines, and  now have zodiac initialize from Game class rather than global variable as it was before. 

Fixes # (issue)

Fix bug where botz explanation wasn't appearing after selecting back button mid game. Add styling so that when resized, the quality matchups arrange into a flex column

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Open index.html, play zodiac game and see that functionality is still working

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +43,7 @@ <h2 id="battleText" class="battle-text">Choose your fighter</h2>
         <div id="signs" class="icons-section"></div>
       </section>
       <section id="botzExplanation" class="botz-explanation">
-        <p class="main-botz-description">Select a zodiac sign to start a game. When a sign is selected, the current Zodiac has the players choice removed - the same thing happens immediately after when the computer is assigned a sign from the pool. Using two moves that are randomly assigned to all signs, a battle between the two begins. The icon, name, element, health points, and quality for each are displayed, along with the users move pool. Each zodiac sign has an associated element (fire, earth, water, or air) and quality (cardinal, muted, or fixed). The matchups are as follows:
+        <p class="main-botz-description">Select a sign and try to conquer the zodiac! Using two moves that are randomly set to all signs, a battle between the player and computer begins. Each zodiac sign has an associated element (fire, earth, water, or air) and quality (cardinal, muted, or fixed). The matchup will determine the advantages at play for a given game. The matchups are as follows:
```
Comment (nicktu12) [inline index.html:46]: Better words 😎

File: game.js [hunk 2]
```diff
@@ -148,21 +143,16 @@ function showBothChoices() {
 }
 
 function selectChoice(event) {
-  console.log("SOMETHING HAPPENING")
     if (event.target.id === "rock") {
       currentUser.currentChoice = "rock"
-      // return "rock"
```
Comment (nicktu12) [inline game.js:154]: nice!

File: main.js [hunk 3]
```diff
@@ -172,14 +174,8 @@ function displayZodiacSelection() {
 }
 
 function updateBattleLog(content) {
-
```
Comment (nicktu12) [inline main.js:175]: The battlelog! A fine addition

File: main.js [hunk 4]
```diff
@@ -302,23 +310,24 @@ function setBoxesAndMoves() {
 
 function newChallenger() {
   if (!currentGame.currentZodiac.length) {
+    debugger
```
Comment (nicktu12) [inline main.js:313]: This will be removed eventually, I hope.

**Review by nicktu12 — APPROVED** [review]
Nice job Bobby! I appreciate a good PR template that is informative.

File: main.js (hunk 4 above)
Comment (hoomberto) [reply main.js:313]: Good catch! I totally missed this, thank you for noticing

**Pull request merged**